include/slurp.h

```diff
@@ -13,6 +13,7 @@
 
 struct slurp_box {
 	int32_t x, y;
+	int32_t output_local_x, output_local_y;
```

Comment:
Hm I'm not a fan of adding these fields to `slurp_box`.

Maybe we can do the computation in `print_formatted_result` instead? Would require to change the arg to something like `slurp_selection` instead of `slurp_box`.